live-examples/wat-examples/numeric/wrap.wat

```diff
@@ -0,0 +1,12 @@
+(module
+  (import "console" "log" (func $log (param i32)))
+  (func $main
+
+    i64.const 10 ;; push an i64 onto the stack
+
+    i32.wrap_i64 ;; wrap from i64 to i32
```

Comment:
"Wrap" may not be a familiar term to everyone, so I suggest giving a brief description of what this does. One can think of it either as reducing the value [mod](https://en.wikipedia.org/wiki/Modular_arithmetic) 2<sup>32</sup>, or as discarding the high 32 bits to produce a value containing just the low 32 bits.

Reply:
Added the explanation to the wrap page in https://github.com/mdn/content/pull/17103 as this pr is just for the example.

I've used almost your exact words as I think it summarises what wrap does really well